crawler.py

```diff
@@ -180,12 +180,13 @@ def url_save(url: str, parent_id: str = None):
     if db_is_url_present(url):
         return
 
-    # 1. add to the db
-    db_add_url(url, parent_id)
+# A runtime independant from main.py, intended to be ran 24/7, performing searches and crawls, then embedding them.
+# Our focus will be on balancing link-crawling, and novel article parsing (news extraction),
+# as well as wikipedia info crawls.
+# All our data will be stored in it's raw form as well, to make it simple to transfer it to another embedding format.
 
-    # 2. place in short-term memory (if there is any space left)
-    if len(url_rapid_queue) < url_queue_limit:
-        url_rapid_queue.append(url)
+# TODO: any moment, user should be able to add new search queries, and perhaps change link/research balance.
+#   example input: ADD news "openai lawsuit"
```

Comment:
what happened here?
looks like you merged and accepted both sides instead of the newer one